src/@types/requests.ts

```diff
@@ -45,6 +45,18 @@ export interface IJoinRoomOpts {
     viaServers?: string[];
 }
 
+export interface IKnockRoomOpts {
```

Comment:
We no longer use the I prefix on interfaces - see https://github.com/vector-im/element-web/blob/develop/code_style.md#typescript--javascript

> Interface names should not be marked with an uppercase I.